packages/jest-cli/src/lib/init/generate_config_file.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import {defaults, descriptions} from 'jest-config';
+
+const generateConfigFile = (results: {[string]: boolean}): string => {
+  const {typescript, coverage, clearMocks, environment} = results;
+
+  const overrides: Object = {};
+
+  if (typescript) {
```

Comment:
I'm not sure if we should add ts-jest - it forces people to install another dependency and babel 7 supports typescript..

@rickhanlonii @thymikee @orta thoughts?

Reply:
Also to the problem with `ts-jest` is that it may happen that older version will be installed, e.g. right after new Jest major release. Maybe we can add another prompt?

Reply:
I feel like this isn't that much of a problem, I've yet to have ts-jest break between jest versions (but I don't see every issue on this repo)  and it's triggered on a prompt that checks your dependencies for typescript itself, so using babel's transformer won't necessarily make sense for that kind of setup by default.